packages/images-plugin/README.md

```diff
@@ -8,10 +8,9 @@ This is an official Appium plugin designed to facilitate image comparison, visua
 2. **Finding Elements by Image** ([docs](./docs/find-by-image.md)) - Using a template image, find a matching screen region of an app and interact with it via standard Appium element semantics.
 
 ## Prerequisites
-
-* [OpenCV](https://opencv.org) 3.x native libraries
-* [opencv4nodejs](https://github.com/justadudewhohacks/opencv4nodejs) NPM module: install with `npm i -g opencv4nodejs`. By default the preinstall script of this module also downloads and makes all the required OpenCV libs from source, but this requires developer tools to be available on the host system.
 * Appium Server 2.0+
+* `@appium/opencv` (installed as part of this image plugin)
```

Comment:
I wouldn't even add this since it will be installed as a dependency; just remove that stuff from the list